### PR TITLE
Magn 9484 unsupported dll format does not generate a proper package manager error message

### PR DESCRIPTION
**Pull request by QilongTang**
### Purpose
http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-9484
Unsupported DLL format does not generate a proper Package Manager error message

### Declarations

Check these if you believe they are true

- [x] The code base is in a better state after this PR
- [x] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate

I do not think we should add corrupted dll to unit test this, but we do have a unit test covering dll import.
- [x] User facing strings, if any, are extracted into `*.resx` files
- [x] All tests pass using the self-service CI.

There are some errors but since my branch is not up to date and I checked errors are not related to my change, I'll open PR to see what happens.
- [x] Snapshot of UI changes, if any.

User will start to get warning after failure to load library. Message will be displayed like this with the actual file path.
warningmsg_v2

### Reviewers

@mjkkirschner @ramramps @sm6srw 

### FYIs

@jnealb

### CODE REVIEW

File: src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs [hunk 1]
```diff
@@ -1979,11 +1980,18 @@ public void ImportLibrary(object parameter)
             DialogResult result = openFileDialog.ShowDialog();
             if (result == DialogResult.OK)
             {
-                foreach (var file in openFileDialog.FileNames)
+                try
```
Comment (ramramps) [inline src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs]: great. can you write a Test around this? It should be tested against UI. May be you can specify any dummy dll and call Import Library. Check whether the exception is caught correctly.

**Comment by mjkkirschner** [comment]
@QilongTang - some thoughts on this PR:

maybe we can add the stack trace or the exception message for the user, there are many reasons why a .dll will not load and some of the exceptions are very useful (locked .dll, Type Load Exception, etc.)

also, when we throw our crash dialog and other dialogs theres a darker color palette that is used and some rectangular buttons in the themes dictionary, potentially consider using that here. Maybe more a question for @Racel .

**Comment by QilongTang** [comment]
@mjkkirschner @Racel Good questions. 
1.For stack trace, I think we need to fold them somehow so that if user are interested, they can unfold to check it, otherwise too much in the single messagebox.
2.For consistant grey look, looks like it is not applied to most of the dialog? Could you give me an example of that? Maybe we need to unify the look of all the pop up dialogs later as well :)

File: src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs (hunk 1 above)
Comment (QilongTang) [inline src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs]: I'll add some testing.

**Comment by mjkkirschner** [comment]
@QilongTang for an example try creating a new preset, or custom node, or double click a node's name title bar. Though these are not warning dialogs, but we also use a similar style for our crash dialog.

**Comment by QilongTang** [comment]
@mjkkirschner Yes, I did notice those. The warning dialog you were talking about is a customized dialog written in WPF so our style is applied. All the other messageboxes living in Dynamo now are windows default messagebox, like the one Zach showed (open file failure) which will have same style with your Windows Theme. I can see your point of making all of them consistant looking with Dynamo but not sure about the priority and size of the task. I'll check with @Racel

**Pull request merged**